.github/ISSUE_TEMPLATE/mystery_event.yml

```diff
@@ -62,11 +75,13 @@ body:
     id: biomes
     attributes:
       label: Biomes
-      description: Select all biomes where the event can occur
+      description: Select all biomes where the event can occur. "ANY, NON-EXTREME, CIVILIZATION and HUMAN are groups of biomes. Check the [Biomes part of the guide](https://github.com/AsdarDevelops/PokeRogue-Events/blob/mystery-battle-events/MEs_Proposal_Guide.md#biomes)."
       multiple: true
       options:
```

Comment:
This could also be changed to checkboxes if you prefer that

Reply:
I think it's fine? Usually it's just one